src/scaffoldmaker/utils/tubemesh.py

```diff
@@ -559,6 +563,14 @@ def createNodesAndElements(region,
         if useCrossDerivatives:
             flatNodetemplate2.setValueNumberOfVersions(flatCoordinates, -1, Node.VALUE_LABEL_D2_DS1DS2, 2)
 
+        flatNodetemplate3 = nodes.createNodetemplate()
```

Comment:
The colon tests don't pass on your branch due to a change in the flat coordinates -- can you please check out.

Reply:
I fixed the test files. The only issue is that when I run the esophagus code, it doesn't create the markers for coordinates and flat coordinates. So, it does not pass the test because of the nodes.getSize(). Could you please have a look?

Reply:
The esophagus tests pass for the main branch. Your branch is a long way behind. Please catch up and test again. If it's still broken, it's almost certainly because of a change you've made e.g. to tubemesh.

Reply:
Done. Fixed the issue.